### PR TITLE
Ember conf redux post

### PR DESCRIPTION
**Pull request by martndemus**

### CODE REVIEW

**Comment by martndemus** [comment]
Still WIP, needs more content and ending

File: source/posts/2016-04-04-emberconf-recap.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+2015's [EmberConf][emberconf] was a conference packed with exciting new things.
+The Ember 2.0 release was closing in. A lot of 1.0s of big Ember projects would
+be shipped soon. [FastBoot][fastboot] was announced and [Glimmer][glimmer] was
+announced. Our new rendering engine was beating React's already blazingly fast
+rendering engine. There was a lot of hype and excitement.
+
+This year the atmosphere of the conference was different. Not a bad different
+persé, but different. This year felt as if we just had achieved these major
```
Comment (poteto) [inline source/posts/2016-04-04-emberconf-recap.md]: per se is 2 words

File: source/posts/2016-04-04-emberconf-recap.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+[Progressive Web Apps][pwa] (_PWA_). With the near release of FastBoot 1.0 and a
+bunch of exciting projects putting effort into pairing up Ember with
+ServiceWorkers and other PWA tools that (recently) have been added to (mobile)
+browsers.
+
+Because of thise developments I would like to dub this year the
+__"Year of Progress and Progressive Web Apps"__.
+
+There is one more thing I want to mention: [Living Styleguides][styleguides].
+Which Chris LoPresto gave a talk about. I did not yet know of this movement
```
Comment (poteto) [inline source/posts/2016-04-04-emberconf-recap.md]: This sentence is a bit awkwardly worded

File: source/posts/2016-04-04-emberconf-recap.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+---
+
+2015's [EmberConf][emberconf] was a conference packed with exciting new things.
+The Ember 2.0 release was closing in. A lot of 1.0s of big Ember projects would
+be shipped soon. [FastBoot][fastboot] was announced and [Glimmer][glimmer] was
+announced. Our new rendering engine was beating React's already blazingly fast
+rendering engine. There was a lot of hype and excitement.
+
+This year the atmosphere of the conference was different. Not a bad different
+per sé, but different. This year felt as if we just had achieved these major
```
Comment (michaeldupuisjr) [inline source/posts/2016-04-04-emberconf-recap.md]: > This year felt as if we just had achieved these major achievements we set out to make at last year's conference and now we are on track to keep leveling up those achievements some more in the foreseeable future.

This year felt as if we had just completed major goals set out at last year's conference, and that we were now on track to keep leveling up those achievements in the foreseeable future.

File: source/posts/2016-04-04-emberconf-recap.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+achievements we set out to make at last year's conference and now we are on
+track to keep leveling up those achievements some more in the foreseeable
+future.
+
+For example Glimmer II drastically improves upon Glimmer I. The 60fps
+rendering demo was super exciting, but it felt different from seeing for the
+first time how comically bad the rendering engine was and then see it one-ups
```
Comment (michaeldupuisjr) [inline source/posts/2016-04-04-emberconf-recap.md]: engine was *before Glimmer `1.0`*

**Comment by michaeldupuisjr** [comment]
Hey @martndemus – I added a couple of comments. In terms of finding a closing for this type of post, you might consider writing about what you think the challenges are as Ember tries to tackle PWA's; or you could talk about what you're most excited to be working on before next year's conference

**Comment by martndemus** [comment]
Thanks @michaeldupuisjr! I think might I just have found an ending!

File: source/posts/2016-04-04-emberconf-recap.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+github: martndemus
+social: true
+summary: "The Year of Progress and Progressive Web Apps"
+published: true
+tags: ember
+---
+
+2015's [EmberConf][emberconf] was a conference packed with exciting new things.
+The Ember 2.0 release was closing in. A lot of 1.0s of big Ember projects would
+be shipped soon. [FastBoot][fastboot] was announced and [Glimmer][glimmer] was
```
Comment (brzpegasus) [inline source/posts/2016-04-04-emberconf-recap.md]: Maybe "FastBoot was announced and so was Glimmer".

File: source/posts/2016-04-04-emberconf-recap.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+author: Marten Schilstra
+twitter: "martndemus"
+github: martndemus
+social: true
+summary: "The Year of Progress and Progressive Web Apps"
+published: true
+tags: ember
+---
+
+2015's [EmberConf][emberconf] was a conference packed with exciting new things.
```
Comment (brzpegasus) [inline source/posts/2016-04-04-emberconf-recap.md]: Maybe you could say "Last year's EmberConf" to emphasize that you did really mean last year. Readers may think you ha vea typo here until they start reading the rest.

File: source/posts/2016-04-04-emberconf-recap.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+The Ember 2.0 release was closing in. A lot of 1.0s of big Ember projects would
+be shipped soon. [FastBoot][fastboot] was announced and [Glimmer][glimmer] was
+announced. Our new rendering engine was beating React's already blazingly fast
+rendering engine. There was a lot of hype and excitement.
+
+This year the atmosphere of the conference was different. Not a bad different
+per sé, but different. This year felt as if we had just completed major goals
```
Comment (brzpegasus) [inline source/posts/2016-04-04-emberconf-recap.md]: "per se" without an accent on the "e"

File: source/posts/2016-04-04-emberconf-recap.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+be shipped soon. [FastBoot][fastboot] was announced and [Glimmer][glimmer] was
+announced. Our new rendering engine was beating React's already blazingly fast
+rendering engine. There was a lot of hype and excitement.
+
+This year the atmosphere of the conference was different. Not a bad different
+per sé, but different. This year felt as if we had just completed major goals 
+set out at last year's conference, and that we were now on track to keep leveling 
+up those achievements in the foreseeable future.
+
+For example Glimmer II drastically improves upon Glimmer I. The 60fps
```
Comment (brzpegasus) [inline source/posts/2016-04-04-emberconf-recap.md]: Comma after "For example"

File: source/posts/2016-04-04-emberconf-recap.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+rendering demo was super exciting, but it felt different from seeing for the
+first time how comically bad the rendering engine was before Glimmer 1.0 and then 
+see it one-ups React's already amazing rendering engine. It was already expected
+that Glimmer II would improve upon Glimmer I.
+
+There was one big new exciting movement this year. That was the movement of
+[Progressive Web Apps][pwa] (_PWA_). With the near release of FastBoot 1.0 and a
+bunch of exciting projects putting effort into pairing up Ember with
+ServiceWorkers and other PWA tools that (recently) have been added to (mobile)
+browsers.
```
Comment (brzpegasus) [inline source/posts/2016-04-04-emberconf-recap.md]: This sentence seems fragmented, unless I'm reading it wrong. It feels like you wanted to say something after "With the near release of FastBoot 1.0..." but the sentence just ends.

Perhaps this is more what you intended:

"There was, however, one big new exciting movement this year: the one of Progressive Web Apps, as highlighted by the near release of FastBoot 1.0 and a bunch of exciting projects putting effort into pairing up Ember with Service Workers and other PWA tools that have recently been added to mobile browsers."

File: source/posts/2016-04-04-emberconf-recap.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+Because of thise developments I would like to dub this year the
```
Comment (brzpegasus) [inline source/posts/2016-04-04-emberconf-recap.md]: `s/thise/these/`

File: source/posts/2016-04-04-emberconf-recap.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+__"Year of Progress and Progressive Web Apps"__.
+
+There is one more thing I want to mention: [Living Styleguides][styleguides].
+Chris LoPresto gave a talk about this. I did not yet know of this movement
+until Luke Melia told me I should go see Chris' talk. It supposed to be
+awesome. I'm glad I followed Luke's advice and went to see Chris' talk. I might
+hear you think: styleguides are for designers, but the way these styleguides are
```
Comment (brzpegasus) [inline source/posts/2016-04-04-emberconf-recap.md]: "You may think that styleguides are for designers" may sound better than "I might hear you think..."

File: source/posts/2016-04-04-emberconf-recap.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+built, they will actually help developers architect better components.
```
Comment (brzpegasus) [inline source/posts/2016-04-04-emberconf-recap.md]: "the way these styleguides are built will actually help developers architect better components"

File: source/posts/2016-04-04-emberconf-recap.md [hunk 13]
```diff
@@ -0,0 +1,60 @@
...
+social: true
+summary: "The Year of Progress and Progressive Web Apps"
+published: true
+tags: ember
+---
+
+Last year's [EmberConf][emberconf] was a conference packed with exciting new things.
+The Ember 2.0 release was closing in. A lot of 1.0s of big Ember projects would
+be shipped soon. [FastBoot][fastboot] was announced and so was [Glimmer][glimmer].
+Our new rendering engine was beating React's already blazingly fast
```
Comment (rsocci) [inline source/posts/2016-04-04-emberconf-recap.md]: Maybe reword this to something along the lines of our engine running at a speed comparable to React's? I'm not sure I 100% like the whole "beating < some-framework >" thing.

File: source/posts/2016-04-04-emberconf-recap.md [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+
+This year the atmosphere of the conference was different. Not a bad different
+per se, but different. This year felt as if we had just completed major goals 
+set out at last year's conference, and that we were now on track to keep leveling 
+up those achievements in the foreseeable future.
+
+For example, Glimmer II drastically improves upon Glimmer I. The 60fps
+rendering demo was super exciting, but it felt different from seeing for the
+first time how comically bad the rendering engine was before Glimmer 1.0 and then 
+see it one-ups React's already amazing rendering engine. It was already expected
```
Comment (maabernethy) [inline source/posts/2016-04-04-emberconf-recap.md]: I sort of agree with Romina. It feels aggressive to keep citing "beating React" as an achievement. I don't think Ember aims to one-up React, I think Ember aims to build amazing software. There is a difference to me

Comment (rsocci) [inline source/posts/2016-04-04-emberconf-recap.md]: Yeah, seems to be 💩 ing on Ember a little too. What if there was no Glimmer? We were still using the framework for what it offers.

File: source/posts/2016-04-04-emberconf-recap.md [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+that Glimmer II would improve upon Glimmer I.
+
+There is one big new exciting movement this year. That is the movement of
+[Progressive Web Apps][pwa] (_PWA_). FastBoot 1.0 will be released soon and 
+there are a bunch of other exciting projects putting effort into pairing up
+Ember with ServiceWorkers and other PWA tools that (recently) have been added
```
Comment (rsocci) [inline source/posts/2016-04-04-emberconf-recap.md]: Either link or explain what a service worker is?

File: source/posts/2016-04-04-emberconf-recap.md [hunk 16]
```diff
@@ -0,0 +1,60 @@
...
+
+For example, Glimmer II drastically improves upon Glimmer I. The 60fps
+rendering demo was super exciting, but it felt different from seeing for the
+first time how comically bad the rendering engine was before Glimmer 1.0 and then 
+see it one-ups React's already amazing rendering engine. It was already expected
+that Glimmer II would improve upon Glimmer I.
+
+There is one big new exciting movement this year. That is the movement of
+[Progressive Web Apps][pwa] (_PWA_). FastBoot 1.0 will be released soon and 
+there are a bunch of other exciting projects putting effort into pairing up
```
Comment (maabernethy) [inline source/posts/2016-04-04-emberconf-recap.md]: The word "exciting"/"excited" is used several times throughout the post. Maybe use a different, more descriptive word in a few places?
